base/linalg/generic.jl

```diff
@@ -514,11 +539,29 @@ julia> norm(v)
 julia> norm(v, Inf)
 6.0
 ```
+"""
+norm(x::AbstractVector, p::Real=2) = vecnorm(x, p)
+
+"""
+    norm(A::AbstractMatrix, p::Real=2)
 
 For matrices, the matrix norm induced by the vector `p`-norm is used, where valid values of
```

Comment:
Here the (more confined) term "matrix norm" appears rather than the (more general) "operator norm", whereas the opposite above. Perhaps use "matrix norm" or "operator norm" consistently? I suppose "matrix norm" is more precise where these methods are concerned.